tests/integration/test_store.py

```diff
@@ -366,7 +366,7 @@ def test_processing(
 @pytest.mark.parametrize(
     "window,max_rate_limit", [(86400, 2 * 86400), (2 * 86400, 86400)]
 )
-@pytest.mark.parametrize("event_type", ["default", "error", "transaction"])
+@pytest.mark.parametrize("event_type", ["default", "error"])
```

Comment:
We still need to run this integration test for transactions, this is still valid behavior.

Reply:
yep, adding this as a new test because modifying this one became too complicated.